homeassistant/components/islamic_prayer_times/coordinator.py

```diff
@@ -38,12 +48,32 @@ def calc_method(self) -> str:
         """Return the calculation method."""
         return self.config_entry.options.get(CONF_CALC_METHOD, DEFAULT_CALC_METHOD)
 
+    @property
+    def lat_adj_method(self) -> str:
+        """Return the latitude adjustment method."""
+        return self.config_entry.options.get(
+            CONF_LAT_ADJ_METHOD, DEFAULT_LAT_ADJ_METHOD
+        ).replace("_", " ")
+
+    @property
+    def midnight_mode(self) -> str:
+        """Return the midnight mode."""
+        return self.config_entry.options.get(CONF_MIDNIGHT_MODE, DEFAULT_MIDNIGHT_MODE)
+
+    @property
+    def school(self) -> str:
+        """Return the school."""
+        return self.config_entry.options.get(CONF_SCHOOL, DEFAULT_SCHOOL)
+
     def get_new_prayer_times(self) -> dict[str, Any]:
         """Fetch prayer times for today."""
         calc = PrayerTimesCalculator(
             latitude=self.hass.config.latitude,
             longitude=self.hass.config.longitude,
             calculation_method=self.calc_method,
+            latitudeAdjustmentMethod=self.lat_adj_method,
```

Comment:
Will the default value for the new three values change the behavior of existing users?

Reply:
These defaults are the same as what defined in the API being used: https://aladhan.com/prayer-times-api
So it won't affect the existing users.